reference/api/settings.md

```diff
@@ -1,61 +1,75 @@
-| **displayedAttributes**  | [Strings] | Fields displayed in the returned documents                                                 | `["*"]` (all attributes)                                                                     |
-| **distinctAttribute**    | String    | Search returns documents with distinct (different) values of the given field               | `null`                                                                                       |
-| **faceting**             | Object    | Faceting settings                                                                          | `{}`                                                                                         |
-| **filterableAttributes** | [Strings] | Attributes to use as [filters and facets](/learn/advanced/filtering_and_faceted_search.md) | `[]`                                                                                         |
-| **pagination**           | Object    | Pagination settings                                                                        | `{}`                                                                                         |
-| **rankingRules**         | [Strings] | List of ranking rules sorted by order of importance                                        | [A list of ordered built-in ranking rules](/learn/core_concepts/relevancy.md#built-in-rules) |
-| **searchableAttributes** | [Strings] | Fields in which to search for matching query words sorted by order of importance           | `["*"]` (all attributes)                                                                     |
-| **sortableAttributes**   | [Strings] | Attributes to use when [sorting](/learn/advanced/sorting.md) search results                | `[]`                                                                                         |
-| **stopWords**            | [Strings] | List of words ignored by Meilisearch when present in search queries                        | `[]`                                                                                         |
-| **synonyms**             | Object    | List of associated words treated similarly                                                 | `{}`                                                                                         |
-| **typoTolerance**        | Object    | Typo tolerance settings                                                                    | `{}`                                                                                         |
+<RouteHighlighter method="GET" route="/indexes/{index_uid}/settings" />
 
-[Learn more about the settings in this guide.](/learn/configuration/settings.md)
+Get the settings of an index. The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.
```

Comment:
Do you think the phrase `The index [uid](..) is required.` is adding anything of value? I feel like it's not and it's repeated 36 times on this page. I'd like to get rid of it. If we wanted, we could add a short paragraph or note box to this effect at the top of the page, since it applies equally to all settings endpoints.

I appreciate the goal of specifying path variables for all endpoints, but from a practical perspective I don't think this is worth the space. The variable is called `index_uid` so I don't see it causing any confusion as long as the user can see the `routeHighlighter` element—and if they can't, we have bigger problems.

Reply:
I looked at other API documentation, and they document path parameters in a lot more detail. But we can get rid of them if we go with [highlighting them](https://github.com/meilisearch/documentation/issues/1808)

Reply:
Indeed, [it is the norm to document path parameters](https://idratherbewriting.com/learnapidoc/docapis_doc_parameters.html#path_parameters), usually in a table. But I remember what we discussed when we removed the tables: the only description for `index_uid` and `task_uid` that we came up with was something like "The index/task UID", which is obviously unhelpful. Also, very few of our endpoints have more than one path parameter, so it was mostly just a bunch of tables with a single row, taking up space without adding information.

Three options:
- We follow the norm and re-implement path parameter tables
- We keep things how they are: a single sentence in the endpoint description
- We try removing the aforementioned sentence and trust our users to find that information themselves (e.g. if you're wondering what `index_uid` is, you click on `indexes` in the sidebar and are greeted with the index object definition—pretty intuitive).

The first option is the most extensible, since it adapts well if we ever have endpoints with many path parameters, or path parameters that are not self-explanatory.

The last option is the most efficient, as it saves space and makes the endpoint descriptions easier to parse.

The middle option (currently implemented) provides no particular benefits IMO. It is not very extensible, and it makes the endpoint descriptions harder to parse.

WDYT @guimachiavelli @maryamsulemani97 ?

Reply:
@dichotommy, what do you think?

Reply:
We'll be using Path parameter tables now, as discussed yesterday